airbyte-webapp/build.gradle

```diff
@@ -49,6 +49,14 @@ task copyDocs(type: Copy) {
     duplicatesStrategy DuplicatesStrategy.INCLUDE
 }
 
+task copyAssets(type: Copy) {
```

Comment:
Could we please leave an inline comment here, for what we're going to need to copy those assets for? Is it for having access to images and assets within the flyout? Would be helpful if this would be documented within this gradle file.